src/components/inputs/number/number.component.tsx

```diff
@@ -10,6 +10,28 @@ import { useTranslation } from 'react-i18next';
 import { useFormProviderContext } from '../../../provider/form-provider';
 import FieldLabel from '../../field-label/field-label.component';
 import { isEmpty } from '../../../validators/form-validator';
+import { Concept } from '@openmrs/esm-framework';
+
+
+const extractFieldUnitsAndRange = (concept?: Concept): string => {
+  if (!concept) {
+    return '';
+  }
+
+  const { hiAbsolute, lowAbsolute, units } = concept;
+  const displayUnits = units ? ` ${units}` : '';  
+  const hasLowerLimit = lowAbsolute != null;
+  const hasUpperLimit = hiAbsolute != null;
```

Comment:
Again `!==`. But I think here it's preferential to just use `isNil()` from lodash.